packages/config/config/devices/0x0312/nzw31t.json

```diff
@@ -1,15 +1,20 @@
 // NIE Technology Co., Ltd. NZW31T
 // In-wall Smart Dimmer
 {
-	"manufacturer": "NIE Technology Co., Ltd.",
+	"manufacturer": "Inovelli",
 	"manufacturerId": "0x0312",
 	"label": "NZW31T",
-	"description": "In-wall Smart Dimmer",
+	"description": "Inovelli In-wall Smart Dimmer",
 	"devices": [
 		{
 			"productType": "0x0111",
 			"productId": "0x261c",
 			"zwaveAllianceId": 2679
+		},
+ 		{
+			"productType": "0x1f02",
+			"productId": "0x1f02",
+			"zwaveAllianceId": 2679
```

Comment:
Don't copy the zwave alliance ID, that's how we track if the device definition came from their files.